analyzers/tests/SonarAnalyzer.Test/Rules/AspNet/BackslashShouldBeAvoidedInAspNetRoutesTest.cs

```diff
@@ -49,16 +49,18 @@ public class BackslashShouldBeAvoidedInAspNetRoutesTest
         [$$$""""[{{{AttributePlaceholder}}}($$"""{{ASlash}}/[action]""")]"""", true, "InterpolatedRawStringLiteralsNotIncludingABackslash"],
     ];
 
-    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.BackslashShouldBeAvoidedInAspNetRoutes>();
-    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.BackslashShouldBeAvoidedInAspNetRoutes>();
+    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.BackslashShouldBeAvoidedInAspNetRoutes>().WithBasePath("AspNet");
+    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.BackslashShouldBeAvoidedInAspNetRoutes>().WithBasePath("AspNet");
 
     private static IEnumerable<object[]> RouteAttributesWithAllTypesOfStrings =>
         AttributesWithAllTypesOfStrings.Select(x => new object[] { ((string)x[0]).Replace(AttributePlaceholder, "Route"), x[1], x[2] });
 
+#if NET
```

Comment:
What about moving this field in the `#if NET` defined [below](https://github.com/SonarSource/sonar-dotnet/pull/9114/files#diff-d8ab6b68e7b3fa1c6843f4a24020c84fc3ca80f110ebdbec377c9867210f9ee4L122)?
It would be closer to the place where it is used, and we would not have multiple `#if NET` intertwined by `#if NET_FRAMEWORK`.